Compilation attempt resulted in this on x86_64 Linux:
```
bsnes/bsnes $  target=libretro binary=library local=false platform=linux make
Compiling target-libretro/libretro.cpp ...
In file included from target-libretro/program.cpp:18,
                 from target-libretro/libretro.cpp:32:
./heuristics/bs-memory.cpp: In member function ‘nall::string Heuristics::BSMemory::manifest() const’:
./heuristics/bs-memory.cpp:29:54: error: expression cannot be used as a function
   29 |   output.append(Memory{}.type("Flash").size(0x100000()).content("Program").text());
      |                                                      ^
../nall/GNUmakefile:177: recipe for target 'obj/libretro.o' failed
make: *** [obj/libretro.o] Error 1
```

--- a/bsnes/heuristics/bs-memory.cpp
+++ b/bsnes/heuristics/bs-memory.cpp
@@ -26,7 +26,7 @@ auto BSMemory::manifest() const -> string {
   output.append("  label:  ", Location::prefix(location), "\n");
   output.append("  name:   ", Location::prefix(location), "\n");
   output.append("  board\n");
-  output.append(Memory{}.type("Flash").size(0x100000()).content("Program").text());
+  output.append(Memory{}.type("Flash").size(0x100000).content("Program").text());
   return output;
 }
 

